src/components/Reports/ReportsPartialsCampaigns/ReportsPartialsCampaigns.js

```diff
@@ -67,105 +68,139 @@ const ReportsPartialsCampaigns = ({ location, dependencies: { dopplerApiClient }
+                      </strong>
+                      {state.campaign.name}
+                    </span>
+                    <span>
+                      <strong>
+                        <FormattedMessage id="reports_partials_campaigns.campaign_subject" />{' '}
+                      </strong>
+                      {state.campaign.subject}
+                    </span>
+                  </S.NameCampign>
                   <span className="arrow" />
                 </div>
               </div>
             </div>
           </header>
           <section className="dp-container">
             <div className="dp-rowflex">
-              <div className="col-sm-12 m-t-24">
-                <p>
-                  <FormattedMessage id="reports_partials_campaigns.campaign_name" />{' '}
-                  <strong>{state.campaign.name}</strong>
-                </p>
-                <p>
-                  <FormattedMessage id="reports_partials_campaigns.campaign_subject" />{' '}
-                  <strong>{state.campaign.subject}</strong>
-                </p>
-              </div>
-            </div>
-            <div className="dp-rowflex">
-              <div className="col-sm-12 m-t-24">
-                <p>
-                  <FormattedMessage id="reports_partials_campaigns.campaign_state" />
-                  <strong>{state.campaignSummaryResults.campaignStatus}</strong>
-                </p>
-                <p>
-                  <FormattedMessage id="reports_partials_campaigns.total_recipients" />{' '}
-                  <strong>{state.campaignSummaryResults.totalRecipients}</strong>
-                </p>
-                <p>
-                  <FormattedMessage id="reports_partials_campaigns.total_sent_so_far" />
-                  <strong>{state.campaignSummaryResults.totalShipped}</strong>
-                </p>
-              </div>
-              <div className="col-sm-12 m-t-24">
-                <h3>
-                  <FormattedMessage id="reports_partials_campaigns.delivery_rate" />
-                </h3>
-                <p>
-                  <strong>{state.campaignSummaryResults.uniqueOpens}</strong>{' '}
-                  <FormattedMessage id="reports_partials_campaigns.opened" />
-                </p>
-                <p>
-                  <strong>{state.campaignSummaryResults.totalUnopened}</strong>{' '}
-                  <FormattedMessage id="reports_partials_campaigns.not_open" />
-                </p>
-                <p>
-                  <strong>
-                    {state.campaignSummaryResults.totalHardBounces +
-                      state.campaignSummaryResults.totalSoftBounces}
-                  </strong>{' '}
-                  <FormattedMessage id="reports_partials_campaigns.hard_and_soft" />
-                </p>
+              <div className="col-sm-12 col-md-12 col-lg-12 m-t-24">
+                <div className="dp-box-shadow">
+                  <S.CampaignMovement>
```

Comment:
Here maybe `S.SecondaryReportBox`? or something in the line of that? as in the meaning that this block comes with secondary or more detailed information.

Reply:
Perhaps `S.DetailedInformationBox` or whatever you can think it would give more meaning and make this reusable in other screens.